extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -401,12 +401,22 @@ object PrivilegesBuilder {
         buildQuery(getQuery, inputObjs)
 
       case "InsertIntoDataSourceDirCommand" |
```

Comment:
Shall we check user's permission to output path? cc @yaooqinn

Reply:
Seems not very necessary, the storage layer often has ACLs of its own.

But, we can do some research first with hive and presto behavior, and make the decision and PR later